scripts/test-data/gen-manual-fixtures.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable @typescript-eslint/unbound-method */
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable @typescript-eslint/no-floating-promises */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+
+const execa = require('execa');
+const fs = require('fs');
+const { resolve } = require('path');
+
+const root = resolve(__dirname, '..', '..');
+const jsonPath = resolve(__dirname, 'data');
+
+/**
+ * gen-manual-fixtures.js
```

Comment:
I think we shouldn't use 'manual' in the name here. There's a lot of existing docs and naming where manual means the opposite of generated content, eg: fixture data that is manually created, rather than generated automatically

Reply:
This is manually created though, it's just copied from a manual JSON to make sure we use the same data in both Storybook and Cypress. It doesn't actually generate it from any source other than a manually created JSON file. 
Is there a preferred term? Maybe `generated-from-manual-fixtures.js` or similar

Reply:
If this script is copying from manual data... why do we need the script? Could we not just use one file?

Reply:
We're copying from JSON because Cypress needs JSON data for fixtures whilst Storybook/Jest needs a `.ts` file. We can't use one or the other because the scripts we use to run this step need to be in CommonJS. So we can't import the manual fixtures that are in Typescript because they use ECMAModules, and I want there to be a single source of truth because updating the JSON files in Cypress is a manual and horrific process. 

Doing it this way we have a single JSON file as a source of truth. When that file is updated the "manual" fixtures that are a Typescript file with a Javascript object exported get updated by converting the JSON file to that format. This is similar to the existing `gen-fixtures.js` step albeit with a manual JSON file rather than one obtained via a `GET ` request. Meanwhile, Cypress gets the JSON file as is but copied over to where it needs to exist.

It's not an ideal solution but it's the simplest one which preserves existing approaches to solving a problem, gives us a single source of truth for shared manual fixtures, and does so using JSON as an accepted way to created a shared data source that is compatible across multiple languages/module systems.